lib/modules/manager/nuget/extract.ts

```diff
@@ -147,3 +149,17 @@ export async function extractPackageFile(
   }
   return res;
 }
+
+export function findVersion(
+  parsedXml: XmlDocument,
+  callback: (arg: XmlElement) => void
+): void {
+  for (const tag of ['Version', 'VersionPrefix']) {
```

Comment:
this should be inside the property group loop, so we don't iterate them multiple times.

Reply:
That changes the semantics of the function. We want to return a value of `Version`, regardless of which `PropertyGroup` it's in, before we look for any `VersionPrefix`. My test wasn't catching this properly, though, so I've made a few more changes.